src/hack/azext_hack/_website_utils.py

```diff
@@ -1,7 +1,7 @@
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
`License.txt` is outdated. The filename is `LICENSE`.

Reply:
There are several style License among our repos, fixing it would be a big project, I will re-format them later.